### ISSUE TITLE
Passphrase & password issues

### ISSUE THREAD

**Issue by skarborg** [arunoda/meteor-up-legacy#48]
Passphrase for my key not accepted (it's def correct), and then I need to enter my password many many times to do anything.

Quite irritating but a very useful tool nonetheless...

**Comment by arunoda** [comment]
Did you set the private key?
You should not use a password for the ssh private key? are you?

---

Arunoda Susiripala
I curate Meteor Weekly - Check it
out!http://meteorhacks.com/meteor-weekly/?utm_source=email-footer&utm_medium=email&utm_campaign=meteorweekly

**Comment by skarborg** [comment]
Yes I did set private key, and yes it has a passphrase. Should I have a key without a passphrase?

**Comment by arunoda** [comment]
yep. It is impossible to work with the passphrase automatically? you need a
key without a passphrase.

---

Arunoda Susiripala

@arunoda http://twitter.com/arunoda
http://gplus.to/arunodahttps://github.com/arunoda
http://www.linkedin.com/in/arunoda

**Comment by skarborg** [comment]
Ok good to know. Please update docs :)

**Comment by arunoda** [comment]
Good Idea. Can you send me a PR?

---

Arunoda Susiripala

@arunoda http://twitter.com/arunoda
 http://gplus.to/arunodahttps://github.com/arunoda
http://www.linkedin.com/in/arunoda

**Comment by skarborg** [comment]
Will try in next few days. Late here now! Thanks Arunoda
